### PR TITLE
Edit Dollar Protocol space

### PR DESCRIPTION
**Pull request by satoshinate**

### CODE REVIEW

**Comment by satoshinate** [comment]
@bonustrack updated Repo to match master

**Review by e-asia — APPROVED** [review]

**Comment by satoshinate** [comment]
any update?

File: spaces/dollar-protocol/index.json [hunk 1]
```diff
@@ -3,8 +3,8 @@
   "symbol": "SHARE",
   "name": "Dollar Protocol",
   "defaultView": "core",
-  "address": "0x6b583cf4aba7bf9d6f8a51b3f1f7c7b2ce59bf15",
-  "token": "0x6b583cf4aba7bf9d6f8a51b3f1f7c7b2ce59bf15",
+  "address": "0x39795344CBCc76cC3Fb94B9D1b15C23c2070C66D",
+  "token": "0x39795344CBCc76cC3Fb94B9D1b15C23c2070C66D",
   "core": [
     "0x89a359A3D37C3A857E62cDE9715900441b47acEC"
   ],
```
Comment (e-asia) [inline spaces/dollar-protocol/index.json:10]: We have updated the format in `index.json`. 
Please add 2 new parameters `chainId` and `decimals`.
You can refer to the documentation that has been completed. https://docs.snapshot.page/guides/create-a-space#3-change-your-space-metadata

**Review by e-asia — CHANGES_REQUESTED** [review]

File: spaces/dollar-protocol/index.json (hunk 1 above)
Comment (satoshinate) [reply spaces/dollar-protocol/index.json:12]: done

**Comment by satoshinate** [comment]
@e-asia fixed now

**Review by e-asia — APPROVED** [review]

**Comment by e-asia** [comment]
@satoshinate If you change the base token address your old proposals will not be visible anymore under your space, are you sure you want to go forward with this?

**Comment by satoshinate** [comment]
@e-asia yes please - we only had 1 test proposal from before. this is now the real deal

**Pull request merged**